packages/forma-36-react-components/src/components/Asset/README.mdx

```diff
@@ -0,0 +1,144 @@
+import { Asset } from './Asset';
+
+# Asset
+
+The Asset component is a visual representation of an asset file such as a video or a PDF document.
+
+<Asset title="Archive of files" type="archive" />
+
+## Table of contents
+
+- [Variations](#variations)
+  - [Archive](#archive)
+  - [Audio](#audio)
+  - [Code](#code)
+  - [Image](#image)
+  - [Markup](#markup)
+  - [PDF](#pdf)
+  - [Plaintext](#plaintext)
+  - [Presentation](#presentation)
+  - [Rich text](#rich-text)
+  - [Spreadsheet](#spreadsheet)
+  - [Video](#video)
+- [Usage](#usage)
+  - [An image asset and an archived image asset](#an-image-asset-and-an-archived-image-asset)
+- [An archive asset](#an-archive-asset)
+
+## Variations
```

Comment:
I would just remove it if we don't have any variations, just for visual reasons.